aws/data_source_aws_iam_group_test.go

```diff
@@ -29,6 +29,33 @@ func TestAccAWSDataSourceIAMGroup_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSDataSourceIAMGroup_member(t *testing.T) {
+	groupName := fmt.Sprintf("test-datasource-group-%d", acctest.RandInt())
+	userName := fmt.Sprintf("test-datasource-user-%d", acctest.RandInt())
+	groupMemberShipName := fmt.Sprintf("test-datasource-group-membership-%d", acctest.RandInt())
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAwsIAMGroupConfigMember(groupName, userName, groupMemberShipName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.aws_iam_group.test", "group_id"),
+					resource.TestCheckResourceAttr("data.aws_iam_group.test", "path", "/"),
+					resource.TestCheckResourceAttr("data.aws_iam_group.test", "group_name", groupName),
+					resource.TestMatchResourceAttr("data.aws_iam_group.test", "arn", regexp.MustCompile("^arn:aws:iam::[0-9]{12}:group/"+groupName)),
```

Comment:
We can simplify this and allow this testing to work in multiple AWS partitions with the below:

```suggestion
					testAccCheckResourceAttrGlobalARN("data.aws_iam_group.test", "arn", "iam", fmt.Sprintf("group/%s", groupName)),
```

Reply:
fixed